packages/cmf/src/actions/http.js

```diff
@@ -27,7 +28,7 @@ function onRequest(url, config) {
 }
 
 function onJSError(error, action) {
-	console.error(error); // eslint-disable-line no-console
+	console.error(error);
```

Comment:
The same way we may want to use Immutable.Map instead of Map to avoid confusion, should we rename this import to something like cmfConsole to prevent the confusion with the actual console ? 
Was just a thought